cartridge/test-helpers/cluster.lua

```diff
@@ -69,9 +70,21 @@ function Cluster:new(object)
 end
 
 function Cluster:initialize()
+    local function merge_server_env(server_config)
+        if self.env then
+            server_config.env = server_config.env or {}
+            for k, v in pairs(self.env) do
+                if server_config.env[k] == nil then
+                    server_config.env[k] = v
+                end
+            end
+        end
+    end
+
     self.servers = {}
     for _, replicaset_config in ipairs(self.replicasets) do
         for i, server_config in ipairs(replicaset_config.servers) do
+            merge_server_env(server_config)
```

Comment:
Purpose of this function is unclear without reading it's body. 
How about something like this or similar for your taste?
```lua
table_supplement(server_config.env, self.env or {})
```

It is already used in argparse module: https://github.com/tarantool/cartridge/blob/47db09d51a0459b375a77ec8f98331ec841adc74/cartridge/argparse.lua#L294-L303

(We could move it to utils and reuse)

Reply:
Let me replace it with `fun.chain(self.env or {}, server_config.env or {}):tomap()`